secured-example's build is failing on the fabric8 booster catalog

The examples should be buildable by `mvn clean install -Popenshift -DskipTests`.

```
[ERROR]   Run 1: BoosterCatalogTest.verifyBoosterCatalog:55 [booster spring-boot/rest-http-secured/current-redhat from https://github.com/snowdrop/secured-example@2.4.9-12-redhat should be buildable by `mvn clean install -Popenshift -DskipTests`] expected:<[0]> but was:<[1]>
```

Executing this statement fails with `The property `SSO_AUTH_SERVER_URL` is missing!` which is a requirement that is included on the `pom.xml` file.

```shell
$ mvn clean install -Popenshift -DskipTests
...

[INFO] 
[INFO] --- maven-clean-plugin:3.1.0:clean (default-clean) @ rest-secured ---
[INFO] Deleting /z/dev/client/redhat/forks-github/secured-example/target
[INFO] 
[INFO] --- maven-enforcer-plugin:3.0.0-M3:enforce (enforce-property) @ rest-secured ---
[WARNING] Rule 0: org.apache.maven.plugins.enforcer.RequireProperty failed with message:
The property `SSO_AUTH_SERVER_URL` is missing!
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  5.857 s
[INFO] Finished at: 2021-12-22T09:40:20+01:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-enforcer-plugin:3.0.0-M3:enforce (enforce-property) on project rest-secured: Some Enforcer rules have failed. Look above for specific messages explaining why the rule failed. -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
```

--- a/pom.xml
+++ b/pom.xml
@@ -159,7 +159,7 @@
                   <message>The property `SSO_AUTH_SERVER_URL` is missing!</message>
                 </requireProperty>
               </rules>
-              <fail>true</fail>
+              <fail>false</fail>
             </configuration>
           </execution>
         </executions>
@@ -196,6 +196,27 @@
                   <goal>integration-test</goal>
                   <goal>verify</goal>
                 </goals>
+              </execution>
+            </executions>
+          </plugin>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-enforcer-plugin</artifactId>
+            <executions>
+              <execution>
+                <id>enforce-property</id>
+                <goals>
+                  <goal>enforce</goal>
+                </goals>
+                <configuration>
+                  <rules>
+                    <requireProperty>
+                      <property>SSO_AUTH_SERVER_URL</property>
+                      <message>The property `SSO_AUTH_SERVER_URL` is missing!</message>
+                    </requireProperty>
+                  </rules>
+                  <fail>true</fail>
+                </configuration>
               </execution>
             </executions>
           </plugin>

